testnets/neutrontestnet/assetlist.json

```diff
@@ -28,6 +28,33 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/neutrontestnet/images/ntrn.svg"
         }
       ]
+    },
+    {
+      "description": "The yield token for ATOM denominated vaults on Amulet Protocol",
+      "denom_units": [
+        {
+          "denom": "amATOM",
```

Comment:
Please swap the denoms (amATOM should be exponent 6, and factory/... should be exponent 0

Reply:
Fixed :+1: